alternator/executor.cc

```diff
@@ -1789,7 +1789,13 @@ future<executor::request_return_type> executor::update_table(client_state& clien
         }
 
         co_await verify_permission(enforce_authorization, client_state_other_shard.get(), schema, auth::permission::ALTER);
-        auto m = co_await service::prepare_column_family_update_announcement(p.local(), schema, std::vector<view_ptr>(), group0_guard.write_timestamp());
+        std::vector<view_ptr> view_updates;
```

Comment:
Nit: you could reserve storage in this vector, we know how long it will be.

Reply:
I moved the new view schema mutations to `prepare_column_family_update_announcement`, so no longer relevant